features/admin.core.v1/store/reducers/config.ts

```diff
@@ -262,6 +262,7 @@ export const commonConfigReducerInitialState: CommonConfigReducerStateInterface<
             productVersionConfig: null,
             selfAppIdentifier: "",
             showAppSwitchButton: undefined,
+            showSmsOtpPasswordRecoveryBetaChip: undefined,
```

Comment:
Let's not use the feature status name in the identifier.

```suggestion
            showSmsOtpPwdRecoveryFeatureStatusChip: undefined,
```